### ISSUE TITLE
[bug] Managed Omni users cannot use `talosctl pcap`

### ISSUE THREAD

**Issue by sneako** [siderolabs/omni-feedback#21]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

```shell
$ talosctl pcap -s 100000 -i eth0 --cluster <CLUSTERNAME> -n <NODENAME> --output talos.pcap
rpc error: code = PermissionDenied desc = not authorized
```
When a user of managed Omni tries to take a packet capture via talosctl, they receive a permission denied error. This is due to the fact that talosconfig that omni generates is locked down right now to avoid operators changing things out from under omni.

### Expected Behavior

Managed Omni users should be able to use `talosctl pcap` on their nodes.

### Steps To Reproduce

1. Create a fully managed Omni cluster
2. Download the talosconfig from the Omni UI
3. Run ` talosctl pcap -s 100000 -i eth0 --cluster <CLUSTERNAME> -n <NODENAME> --output talos.pcap --talosconfig <TALOSCONFIG_PATH>`

**Comment by smira** [comment]
The fix will be released as part in Omni v0.7.0+

**Comment by sneako** [comment]
Omni v0.7.0 has been released and I can confirm that I am now able to take pcaps via talosctl, thanks!
